src/ScClient.ts

```diff
@@ -15,6 +15,12 @@ export interface Response<T = any> {
   status: boolean;
   event: boolean;
   payload: T;
+  errors: string | Error[];
+}
+
+export interface Error {
```

Comment:
Lets name it like ScError or BackendError, because Error interface is already present in DOM so is may be confusing